{"name": "Создание заказа неавторизованным пользователем", "status": "failed", "statusDetails": {"message": "AssertionError: assert (200 == 401)\n +  where 200 = <Response [200]>.status_code\n +  and   401 = StatusCodes.UNAUTHORIZED", "trace": "self = <test_order_create.TestOrderCreate object at 0x00000230DA30A640>\n\n    @allure.title('Создание заказа неавторизованным пользователем')\n    @allure.description('Проверка на ошибку, соответствие статус-кода и ответа')\n    def test_order_create_not_auth_user(self):\n        response = API.order_create(data=Other.SUCCESS_INGRIDIENTS_DATA, headers=None)\n>       assert response.status_code == StatusCodes.UNAUTHORIZED and response.json()[Other.MESSAGE_JSON] == Responses.NOT_ALLOWED_AUTH\nE       assert (200 == 401)\nE        +  where 200 = <Response [200]>.status_code\nE        +  and   401 = StatusCodes.UNAUTHORIZED\n\ntests\\test_order_create.py:19: AssertionError"}, "description": "Проверка на ошибку, соответствие статус-кода и ответа", "start": 1727014228423, "stop": 1727014228652, "uuid": "a7db614b-ead6-42df-b241-a4e867699d15", "historyId": "f0130ba18d9354eff6c85e2f1e5c19fe", "testCaseId": "f0130ba18d9354eff6c85e2f1e5c19fe", "fullName": "tests.test_order_create.TestOrderCreate#test_order_create_not_auth_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_create"}, {"name": "subSuite", "value": "TestOrderCreate"}, {"name": "host", "value": "FanatovPC"}, {"name": "thread", "value": "14972-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_create"}]}